### ISSUE TITLE
bug : Design Doc link not accessible 

### ISSUE THREAD

**Issue by kingrishabdugar** [CodeForGoodTech/C4GT#2]
The Design Doc can be found [here](https://docs.google.com/document/d/1aX9OL_LKnkAojXyfvYLjSxXoUnhlYMjVkO9Obr0Fh0E/edit).

**Comment by kingrishabdugar** [comment]
Please make the google doc accessible for everyone in the read-only mode without editor access for the project :https://github.com/Code4GovTech/C4GT/wiki/Doc-Generator

**Comment by amankumar003** [comment]
@kingrishabdugar can we start with good first issues, so that we can add this issue link and discription while making the proposal

**Comment by amankumar003** [comment]
@kingrishabdugar last date for proposal submission is 11th of june, is submitting proposal at early, benefits or increases the chances of proposal acceptance ?
should I submit the proposal now or wait and understand and make some contribution and then submit the proposal ?

**Comment by kingrishabdugar** [comment]
> @kingrishabdugar can we start with good first issues, so that we can add this issue link and discription while making the proposal

I don't think that this qualifies for a "good first issue" because it is more of a bug which we cannot fix anyhow since we don't have the access to the doc file

Submitting proposals early might have an edge but owing to my limited understanding better proposal submitted at the last moment might have an edge over an early proposal with limited contributions
